vms/volunteer/templates/volunteer/profile.html

```diff
@@ -1,121 +1,121 @@
-                </div>
-            {% endif %}
-            {% if volunteer.resume_file %}
-                <div class="panel-group" id="accordion_resume_file">
-                    <div class="panel panel-default">
-                        <div class="panel-heading">
-                            <h4 class="panel-title">
-                                <a data-toggle="collapse" data-parent="#accordion_resume_file" href="#collapseResumeFile">
-                                    {% trans "Uploads" %}
-                                </a>
-                            </h4>
-                        </div>
-                        <div id="collapseResumeFile" class="panel-collapse collapse in">
-                            <div class="panel-body">
-                                <form action="{% url 'volunteer:download_resume' volunteer.id %}" method="post">
-                                    {% csrf_token %}
-                                    <button class="btn btn-default" type="submit">{% trans "Download Resume" %}</button>
-                                </form>
-                            </div>
-                        </div>
-                    </div>
-                </div>
-            {% endif %}
-            <div class="panel panel-default">
-                <div class="panel-heading">
-                    <h4 class="panel-title">
-                        {% trans "Options" %}
-                    </h4>
-                </div>
-                <div class="panel-body">
-                    <a href="{% url 'volunteer:edit' volunteer.id %}" class="btn btn-default">{% trans "Edit Profile" %}</a>
-                </div>
-            </div>
-        </div>
-    </div>
-{% endblock %}
+{% extends "vms/base.html" %}
+
+{% load i18n %}
+
+{% block content %}
+    <div class="spacer"></div>
+    <div class="row">
+        <h1>
+            {{ user.volunteer.first_name }} {{ user.volunteer.last_name }}
+        </h1>
+        <hr>
+        <div class="col-md-12">
+            <div class="panel-group" id="accordion_desc">
+                <div class="panel panel-default">
+                    <div class="panel-heading">
+                        <h4 class="panel-title">
+                            <a data-toggle="collapse" data-parent="#accordion_desc" href="#collapseDesc">
+                                {% trans "About Me" %}
+                            </a>
+                        </h4>
+                    </div>
+                    <div id="collapseDesc" class="panel-collapse collapse in">
+                        <div class="panel-body">
+                            {{ volunteer.description }}
+                        </div>
+                    </div>
+                </div>
+            </div>
+            <div class="panel panel-default">
+                <div class="panel-heading">
+                    <h4 class="panel-title">
+                        {% trans "Details" %}
+                    </h4>
+                </div>
```

Comment:
seems like some editor issue, can you please look into what is causing everything to come as a diff, it'll be easier to look.